src/neo4j-arc/cypher-language-support/cypher-editor/CypherEditor.tsx

```diff
@@ -57,44 +55,68 @@ const MonacoStyleWrapper = styled.div`
 const EXPLAIN_QUERY_PREFIX = 'EXPLAIN '
 const EXPLAIN_QUERY_PREFIX_LENGTH = EXPLAIN_QUERY_PREFIX.length
 const EDITOR_UPDATE_DEBOUNCE_TIME = 300
-type MonacoDefaultProps = {
-  value: string
-  onDisplayHelpKeys: () => void
-  onChange: (value: string) => void
-  onExecute: (value: string) => void
-}
-
-export type MonacoProps = MonacoDefaultProps & {
+type CypherEditorDefaultProps = {
```

Comment:
There is a Required util in typescript, which converts optionals to required, so suggest to ``type CypherEditorDefaultProps = Required<CypherEditorProps>`` to not have it defined twice.